### PR TITLE
Add variables for pushing configs to clients for DNS

### PR DESCRIPTION
**Pull request by felddy**
# 

## 🗣 Description

Adds to variables that are inserted into the OpenVPN configuration template. 
They send the client the DNS server to use as well as the search domain.

## 💭 Motivation and Context

Clients need DNS yo!

## 🧪 Testing

Deployed to production successfully.

## 🚥 Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -15,6 +15,14 @@ variable "cert_read_role_accounts_allowed" {
   default     = []
 }
 
+variable "client_dns_search_domain" {
+  description = "The DNS search domain to be pushed to VPN clients."
```
Comment (jsf9k) [inline variables.tf:20]: Can these new variables be specified as type `string`?

**Review by jsf9k — CHANGES_REQUESTED** [review]
One minor request, then I will approve.

File: variables.tf (hunk 1 above)
Comment (felddy) [reply variables.tf:20]: dun: 8e78b64

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
Rock and roll! 🎸

**Pull request merged**